x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/decreased_throughput_jsbase.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM JSBase: Identifies periods during which the application is processing fewer requests than normal.",
+  "analysis_config": {
+    "summary_count_field_name": "doc_count",
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "low_count",
+        "function": "low_count"
+      }
+    ],
+    "influencers": [
+      "service.name"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "custom_urls": [
```

Comment:
This is missing a `created_by` property - it should be `"created_by": "ml-module-apm-jsbase",`

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69